src/lib.rs

```diff
@@ -1,18 +1,127 @@
+//! # Diesel dynamic schema
+//!
+//! Diesel is an ORM and query builder designed to reduce 
+//! the boilerplate for database interactions.
+//!
+//! If this is your first time reading about Diesel, then
+//! we recommend you start with the [getting started guide].
+//! We also have [many other long form guides].
+//!
+//! [getting started guide]: https://diesel.rs/guides/getting-started/
+//! [many other long form guides]: https://diesel.rs/guides
+//!
+//! Diesel is built to provide strong compile time guarantees that your
+//! queries are valid. To do this, it needs to represent your schema 
+//! at compile time. However, there are some times where you don't 
+//! actually know the schema you're interacting with until runtime.
+//! 
+//! This crate provides tools to work with those cases, while still being
+//! able to use Diesel's query builder. Keep in mind that many compile time 
+//! guarantees are lost. We cannot verify that the tables/columns you ask 
+//! for actually exist, or that the types you state are correct.
+//! 
+//! # Getting Started
+//! 
+//! The main function used by this crate is `table`. Note that you must always
```

Comment:
I would like to word that sentence a bit different, because we may add more functionality here "soon". Maybe something like
"The `table` functions is used to …"

Reply:
Doing.